### PR TITLE
Added ability to specify network in config object

### PR DESCRIPTION
**Pull request by kkanzelmeyer**

### CODE REVIEW

**Comment by kkanzelmeyer** [comment]
The dependency updates fix the conflicts with versions of node > 4.3.0. I ran the unit tests with node 6.5.0 and they all passed.

File: bigio/utils.js [hunk 1]
```diff
@@ -100,42 +100,42 @@ module.exports = {
...
-                for(var intfc in interfaces) {
-                    if(intfc.indexOf(match) > -1) {
-                        for(var i in interfaces[intfc]) {
-                            if(interfaces[intfc][i].family == 'IPv4') {
-                                ip = interfaces[intfc][i].address;
-                            }
-                        }
-                    }
-                }
+    getIp: function (cb) {
```
Comment (andy-trimble) [inline bigio/utils.js:103]: The indent of this function is only two spaces, while the rest of the file is four. Please either increase to four, or decrease the rest of the file to two spaces to maintain consistency.

**Review by andy-trimble — CHANGES_REQUESTED** [review]
Just a formatting change requested, looks good otherwise.

File: bigio/utils.js (hunk 1 above)
Comment (kkanzelmeyer) [reply bigio/utils.js:103]: done, thanks!

**Comment by kkanzelmeyer** [comment]
To maintain project consistency I used four spaces for the indentation. Thanks!

**Review by andy-trimble — APPROVED** [review]

**Pull request merged**